internal/controller/vrg_kubeobjects.go

```diff
@@ -495,76 +495,54 @@ func (v *VRGInstance) kubeObjectsCaptureStatus(status metav1.ConditionStatus, re
 	}
 }
 
-func (v *VRGInstance) kubeObjectsRecover(result *ctrl.Result,
-	s3StoreProfile ramen.S3StoreProfile, objectStorer ObjectStorer,
-) error {
-	if v.kubeObjectProtectionDisabled("recovery") {
-		return nil
-	}
+func (v *VRGInstance) getVRGFromS3Profile(s3ProfileName string) (*ramen.VolumeReplicationGroup, error) {
```

Comment:
We have this function: `GetLastKnownVRGPrimaryFromS3`. Can we have a common function?

Reply:
@BenamarMk that is a function in DRPC and this is a function in VRG as a method on VRGInstance, can I take up the refactoring in a later PR?